### ISSUE TITLE
Technical testing

### ISSUE THREAD

**Issue by adswa** [adswa/mpi-datamanagement-ws#20]
We should schedule a test run for:

- Video conferencing system (I've seen issues today on BigBlueButton, so we should test, and migrate if needed)
- Code execution tool (if @lnnrtwttkhn wants to use it, too)

**Comment by lnnrtwttkhn** [comment]
Blue Button worked very well!

We can have:
- flexible screen sharing
- drawings on slides
- captions
- breakout rooms

I think that's pretty much all we'll need.

Did you find the recording of our test session @adswa?

**Comment by adswa** [comment]
Yes, I found it: https://webconf.fz-juelich.de/playback/presentation/2.0/playback.html?meetingId=4b993afbf525645198ea22126d4626099dcafad0-1604053903909

**Comment by adswa** [comment]
Phew, fixed the Github workflow failure in b0a88eaa5710b47ff89eab4 :)
